atf_eregs/templates/regulations/landing_479.html

```diff
@@ -36,5 +68,13 @@ <h3 class="list-header">The regulation covers topics such as:</h3>
     <li><a href="https://www.atf.gov/resource-center/forms-library"
            class="external" aria-label="Link opens in a new window" target="_blank">NFA Forms</a></li>
   </ul>
+  
+  <h3>Related tools</h3>
+  <ul>
+    <li>View this regulation on <a href="http://www.ecfr.gov/cgi-bin/text-idx?SID=3348fa35fa821aaaff8cd8d898a5dedd&mc=true&amp;node=pt27.3.479&amp;rgn=div5" class="external" target="_blank">e-CFR (Electronic Code of Federal Regulations)</a></li>
+    <li>Related rules, proposed rules, and notices in the <a href="https://www.federalregister.gov/articles/search?conditions%5Bagency_ids%5D%5B%5D=19&conditions%5Bterm%5D=Machine+Guns%2C+Destructive+Devices%2C+and+Certain+Other+Firearms&amp;order=newest&amp;quiet=true" class="external" target="_blank">Federal Register</a></li>
+    <li>Related open and closed comment periods on <a href="http://www.regulations.gov/#!searchResults;rpp=25;so=DESC;sb=commentDueDate;po=0;s=Machine%252BGuns%252C%252BDestructive%252BDevices%252C%252Band%252BCertain%252BOther%252BFirearms;fp=true;dct=FR%252BPR%252BN%252BO;a=ATF" class="external" target="_blank">Regulations.gov</a></li>
```

Comment:
Are these missing aria-labels? Also, we should encode the ampersands to be compliant